File: eFormAPI/Plugins/RentableItems.Pn/RentableItems.Pn/Infrastructure/Models/ContractInspectionsRequestModel.cs
namespace RentableItems.Pn.Infrastructure.Models
{
    public class ContractInspectionsRequestModel
    {
        public string ContractId { get; set; }
        public string SortColumnName { get; set; }
        public int Offset { get; set; }
        public int PageSize { get; set; }
        public bool IsSortDsc { get; set; }
    }
}

File: eform-client/src/app/plugins/modules/rentable-items-pn/models/contract-inspections-request.model.ts
export class ContractInspectionsRequestModel {
  sortColumnName: string;
  pageIndex: number;
  offset: number;
  pageSize: number;
  isSortDsc: boolean;

  constructor() {
    this.sortColumnName = 'Id';
    this.isSortDsc = true;
    this.pageSize = 10;
    this.pageIndex = 1;
    this.offset = 0;
  }
}
